web/modules/custom/ys_base/js/ys_base.js

```
@@ -0,0 +1,76 @@
+/**
+ * @file
+ * YS Base module JavaScript behaviors.
+ */
+
+(function (Drupal) {
+  'use strict';
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)